optuna/visualization/_slice.py

```diff
@@ -21,6 +24,82 @@
 _logger = get_logger(__name__)
 
 
+class _SliceSubplotInfo(NamedTuple):
+    param_name: str
+    x: List[Any]
+    y: List[float]
+    trial_numbers: List[int]
+    is_x_log_scale: bool
+    is_x_nunmerical: bool
```

Comment:
For consistency with https://github.com/optuna/optuna/pull/3734/files#diff-fb4c806a38d57b2092f804687882f64ad7ad51616bb06d183acdd72acee671d4R40-R41
```suggestion
    is_log: bool
    is_cat: bool
```

Reply:
`is_x_numerical` is different from `not is_cat`.

Reply:
Thanks. Then, `is_numerical` is fine?

Reply:
Do you have an idea of short name for `is_x_numerical`? (Do you think `is_num` will do?)